### ISSUE TITLE
unexpected "[strong]"  tag in LaTeX output for C++ class enums.

### ISSUE THREAD

**Issue by calumblair** [doxygen/doxygen#6599]
test.h:
```
/** 
enum class description
*/
enum class myclass
{
	value1 = 1, ///< representation of value 1
	value2 = 2, ///< representation of value 2
};
```

Expected pdflatex output:
```
myclass
enum myclass
enum class description
```
Observed pdflatex output:
```
myclass
enum myclass [strong]
enum class description
```

The "[strong]" tag is not present in the HTML output, so it looks like it's only the LaTeX.
MWE attached.
[doxygen_unexpected_strong.zip](https://github.com/doxygen/doxygen/files/2548842/doxygen_unexpected_strong.zip)

**Comment by albert-github** [comment]
I had a quick look and with the HTML output I also see the 'strong' though a little bit differently depicted:

![strong_html](https://user-images.githubusercontent.com/5223533/48005093-db514f00-e112-11e8-82e0-9e555ceb0207.png)


So looks like a small misinterpretation.

**Comment by AleixFerre** [comment]
It happens in all C variants (C, C++ and C#)

![image](https://user-images.githubusercontent.com/12767963/128850978-0100c22c-c34e-40d1-a5e2-66b3c7e50496.png)

**Comment by albert-github** [comment]
I think the output of the original is correct as the `enum` is a `enum class` so strongly typed and is now correct(in the doxygen 1..1 version).

@AleixFerre  You specified that  the problem also appears for a number of other languages. 
- Can you please attach a, small, self contained example (source+configuration file in a tar or zip) that shows the problem that should be fixed with the proposed patch? Please don't add external links as they might not be persistent. 
- Please also specify the full doxygen version used (`doxygen -v`).

**Comment by AleixFerre** [comment]
With the image above, the corresponding code is (in the `Scenes.cs` file)
```cs
/// <summary>
/// Alias for all the scenes in the game. Use this enum instead of the actual number index or name string.
/// This will be maintained through the development of the game, the literal strings or ints, will not.
/// </summary>
public enum Scenes {
    LOADING = 0, ///< The Connect to server scene. This is the first you see at the start of the game
    MANAGER = 1, ///< The Async persistent Manager scene that is running all the time in the background
    MAIN_MENU = 2, ///< The main menu scene
    LOBBY = 3, ///< The lobby scene. This is where you connect to a room and select the character.
    INGAME = 4, ///< The in-game scene. Here you play the actual game
    POST_GAME = 5 ///< The post-game scene. Here you see a resume of the past game.
}
```

I'm using Unity to develop this game, and the doxygen version is
```sh
$ doxygen -v
1.9.1 (ef9b20ac7f8a8621fcfc299f8bd0b80422390f4b)
```

What I don't get is what the strong tag means. I don't know if it's what it should show or not.

**Comment by AleixFerre** [comment]
And the compiler does not like this

![image](https://user-images.githubusercontent.com/12767963/128879805-d490ddd3-9b73-4bde-8d57-9856cc90ef93.png)

**Comment by albert-github** [comment]
There is difference between C++ and CSharp. In CSharp all classes are by default strong. For C++, in the past, there was only the C `enum` construct, in C++ later on the construct `enum class` has been introduced and to give a visual difference the label `strong` has been placed. In C++ a `enum class` is also called a a strong typed enum.
(all by head)

For CSharp all `enum`s are by definition strong typed (when I'm not mistaken) so the label `strong` might be a bit redundant.

@doxygen 
- did I misinterpret / miss something?
- should the label `strong` be removed for CSharp? (might by a bit tricky to do this in the code).

**Comment by doxygen** [comment]
Strong should indeed not be shown in C#. I've made a fix for this.

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.2.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
